tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -1,135 +1,37 @@
-    > - Serverless Tier clusters only support TLS connection. For more information, see [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
-
-</div>
-
-<div label="Dedicated Tier">
-
-To connect to a Dedicated Tier cluster via standard connection, perform the following steps:
-
-1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.
-
-2. Locate your cluster, and click **Connect** in the upper-right corner of the cluster area. A connection dialog box is displayed.
-
-3. Create a traffic filter for the cluster. Traffic filter is a list of IPs and CIDR addresses that are allowed to access TiDB Cloud via a SQL client.
-
-    If the traffic filter is already set, skip the following sub-steps. If the traffic filter is empty, take the following sub-steps to add one.
-
-    1. Click one of the buttons to add some rules quickly.
+## Serverless Tier
 
-        - **Add My Current IP Address**
-        - **Allow Access from Anywhere**
+For Serverless Tier clusters, you can connect to your cluster via a SQL client or via SQL editor in the TiDB Cloud console.
 
-    2. Provide an optional description for the newly added IP address or CIDR range.
+- [Connect via standard connection](/tidb-cloud/connect-via-standard-connection.md#serverless-tier)
 
-    3. Click **Create Filter** to confirm the changes.
+    The standard connection exposes a public endpoint with traffic filters, so you can connect to your TiDB cluster from your laptop.
 
-4. Under **Step 2: Download TiDB cluster CA** in the dialog, click **Download TiDB cluster CA** for TLS connection to TiDB clusters. The TiDB cluster CA supports TLS 1.2 version by default.
+    Serverless Tier only [supports TLS connections](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md), which ensures the security of data transmission from your applications to TiDB clusters.
 
-    > **Note:**
-    >
-    > - The TiDB cluster CA is only available for Dedicated Tier clusters.
-    > - Currently, TiDB Cloud only provides the connection strings and sample code for these connection methods: MySQL, MyCLI, JDBC, Python, Go, and Node.js.
+- Connect via SQL editor
 
-5. Under **Step 3: Connect with a SQL client** in the dialog, click the tab of your preferred connection method, and then refer to the connection string and sample code on the tab to connect to your cluster.
+    SQL editor is a web-based SQL editor that allows you to directly edit and run SQL queries against databases of Serverless Tier. To access SQL editor, in the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, locate your cluster, click **Connect** and then select **SQL Editor** in the drop-down list.
 
```

Comment:
Btw, although accessing SQL editor is simple, it is still worth creating a separate doc to promote it to users, just as "Connect via SQL Shell".

Reply:
Maybe we can wait until the SQL editor is GA? There might be drastic changes to the UI and features before then.

Reply:
+1